utils/build-script

```diff
@@ -758,16 +758,17 @@ details of the setups of other systems or automated environments.""")
              "implementation for Android builds",
         metavar="PATH")
     android_group.add_argument(
-        "--android-ndk-version",
-        help="A version of the NDK to use when building for Android. "
+        "--android-api-level",
+        help="The Android API level to target when building for Android. "
              "Currently only 21 or above is supported",
         default="21")
     android_group.add_argument(
         "--android-ndk-toolchain-version",
         help="A version of the toolchain to use when building for Android. "
-             "Use 4.8 for 32-bit builds, 4.9 for 64-bit builds",
+             "Use 4.8 for 32-bit builds, 4.9 for 64-bit builds. "
+             "%(default)s by default",
         choices=["4.8", "4.9"],
-        default="4.8")
```

Comment:
Ah yes, that's definitely something to consider. Perhaps this shouldn't be an option at all... but then again, having this option made it easy to provide a short term fix for https://bugs.swift.org/browse/SR-1264 -- one part of the solution was passing 4.9 instead of 4.8.

I don't have strong feelings one way or the other. I could remove the 4.8 option, but personally don't mind it being there.